pkg/apis/crd/v1beta1/types.go

```diff
@@ -692,6 +692,9 @@ const (
 	// RuleActionReject indicates that the traffic matching the rule must be rejected and the
 	// client will receive a response.
 	RuleActionReject RuleAction = "Reject"
+	// RuleActionIsolation indicates that the traffic matching the rule should be
+	// affected by Kubernetes NetworkPolicy default isolation.
+	RuleActionIsolation RuleAction = "Isolation"
```

Comment:
Trying to understand this. Is this a real action or for other purpose? I feel it sounds a bit confusing. The existing actions are all verbs clearly stating what action will be taken, while "Isolation" describes a state. And I don't get its difference from "Pass".

Reply:
No I did not mean this should be a real action. And I agree that it does not make sense to be added to the CRD action type here. @qiyueyao what we want is simply a "isolate" string in the `networkpolicyevaluation` result if it is an isolation rule

Reply:
Understood, one question: if we want this "isolation" also in the API response, or just in the antctl response?
Currently added a step to identify default isolation rules in antctl processing, let me know if we also want it in API response (the place adding a real action, but a string instead of action constant).